### DISCUSSION TITLE
other simpleton questions before i go away.

### DISCUSSION THREAD

**Discussion by hiqsociety** [gstrauss/mcdb#13] [General]
1. in what scenario is mcdb being used or designed for? ( i know this is like an improved version of cdb but... for modern applications... more for embedded devices? just curious )

for "constant database", i guess the comparison is most aptly compared with lmdb? i have yet to see any actual benchmark data comparing either side by side

2. how can i check on the use case of the production users and their feedback etc? i'm curious because after 12 years, there's only 103 stars and 12 issue questions in 12 years.

merry xmas by the way.

**Comment by gstrauss** [comment]
Title:
>  "other simpleton questions before i go away."

Sorry.  For someone brazen enough to name their github account "hiqsociety", you do not seem to be able to exercise and apply it to writing an intelligent subject/title line, even after having been criticized in #11.  I have concluded that it would be a waste of time for me to try to have any "conversation" with you.
